resources/prometheus/unit_tests/RHACSFleetschardCertificateExpiring.yaml

```diff
@@ -0,0 +1,51 @@
+rule_files:
+  - /tmp/prometheus-rules-test.yaml
+
+evaluation_interval: 1m
+
+tests:
+  - interval: 1d
+    input_series:
+      - series: acs_fleetshard_certificate_expiration_timestamp{exported_namespace="rhacs", secret="secret", data_key="key"}
```

Comment:
Can you change the exported_namespace to `rhacs-00000000000000000000` so that it's clear it corresponds to a tenant namespace?

Reply:
Done, hopefully that is what was meant